test/controllers/IndexControllerSpec.scala

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers
+
+import base.{MockNunjucksRendererApp, SpecBase}
+import config.FrontendAppConfig
+import matchers.JsonMatchers
+import org.mockito.ArgumentMatchers.any
+import org.mockito.Mockito.{reset, when}
+import org.scalatestplus.mockito.MockitoSugar
+import play.api.inject.bind
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import play.twirl.api.Html
+
+import scala.concurrent.Future
+
+class IndexControllerSpec extends SpecBase with MockitoSugar with JsonMatchers with MockNunjucksRendererApp {
+
+  val mockFrontendAppConfig = mock[FrontendAppConfig]
```

Comment:
Is this actually needed? I don't see it actually having a mock set on it.

Reply:
Yea it's not required in there and also no longer needed in the RedirectControllerSpec, I'll remove from both